libs/platform/ebpf_epoch.c

```diff
@@ -40,6 +40,11 @@
 // Stale flag:
 // The stale flag is set if the timer runs and the ebpf_epoch_cpu_entry_t has entries in its free list.
 // If the stale flag is already set, then the per-CPU stale_worker is scheduled.
+//
```

Comment:
please update the epoch md and also mention about the cpu table, thread table and the reserved epoch.